src/hotspot/cpu/aarch64/aarch64_sve.ad

```diff
@@ -168,8 +169,15 @@ source %{
         break;
     }
     // By default, we only support vector operations with no less than 8 bytes and 2 elements.
-    return 8 <= length_in_bytes && length_in_bytes <= MaxVectorSize && vlen >= 2;
+    // TODO: set min vector size to 8 and have a full test.
+    return 16 <= length_in_bytes && length_in_bytes <= MaxVectorSize && vlen >= 2;
```

Comment:
I think we should fix this TODO in this patch.

Reply:
Fixed in this patch. Thanks for the advice!